### PR TITLE
Replace ToDirection & ToCoordinate with Into

### PR DESCRIPTION
**Pull request by serprex**
Perhaps undesirable given type inference confusion

Also remove `to_pixel_float`

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -828,13 +819,13 @@ impl<I : Integer> Coordinate<I> {
             CCW(d) => (LeftBack, Left, d),
         };
 
-        let mut cur_coord = *self + start_dir.to_coordinate().scale(
+        let mut cur_coord = *self + Into::<Coordinate<_>>::into(start_dir).scale(
```
Comment (dpc) [inline src/lib.rs]: Wouldn't `Coordinate::from(start_dir)` be shorter?

File: src/lib.rs [hunk 2]
```diff
@@ -828,13 +819,13 @@ impl<I : Integer> Coordinate<I> {
...
             num::FromPrimitive::from_i32(r).unwrap()
-            );
+        );
         let mut cur_dir = start_dir + start_angle;
 
         for _ in 0..6 {
-            let cur_dir_coord = cur_dir.to_coordinate();
+            let cur_dir_coord: Coordinate<_> = cur_dir.into();
```
Comment (dpc) [inline src/lib.rs]: Again, `T::from` seems more natural.

File: src/lib.rs [hunk 3]
```diff
@@ -192,6 +180,15 @@ pub enum Spin {
     CCW(Direction),
 }
 
+impl Into<Direction> for Spin {
```
Comment (dpc) [inline src/lib.rs]: Why not `From<Spin> for `Direction`? https://doc.rust-lang.org/std/convert/trait.Into.html "Library authors should not directly implement this trait, but should prefer implementing the From trait (...)"?

Comment (serprex) [reply src/lib.rs]: Thank you, my bad, will revise

**Pull request merged**